tap-setup-scripts/README_FIRST.md

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Temporary README file for internal TAP user's reference. This is not exposed to customers